Doc/howto/regex.rst

```diff
@@ -230,13 +230,13 @@ while ``+`` requires at least *one* occurrence.  To use a similar example,
 ``ca+t`` will match ``'cat'`` (1 ``'a'``), ``'caaat'`` (3 ``'a'``\ s), but won't
 match ``'ct'``.
 
-There are two more repeating qualifiers.  The question mark character, ``?``,
+There are two more repeating operators or quantifiers.  The question mark character, ``?``,
```

Comment:
Wrap at 80 chars.

Reply:
I think that in this case it is better to keep a line slightly longer that 80 characters than split it on two very short lines or reformat the whole paragraph. PEP 8 allows some liberty.

Reply:
I wrongly assumed that in Python docs (as we do in Django docs) re-wrapping is stricter. TIL. Thanks :+1:

Reply:
Small compromises are possible. If it would be a new text, it would be stricter.